bga_database/static/js/chart_helper.js

```diff
@@ -167,9 +182,15 @@ var ChartHelper = {
       chart: {
         type: 'pie'
       },
+      colors: ['#004C76', '#538BC2'],
       title: {
         text: ''
       },
+      tooltip: {
+        formatter: function() {
+          return this.key + ": $" +  this.y;
```

Comment:
This looks awesome! One thing: Can you add commas to the number? Thankfully, [it's pretty simple](https://github.com/datamade/bga-payroll/blob/7209d327164ad808f7e0493b088423ce7915f54c/bga_database/static/js/chart_helper.js#L25)!

Reply:
I knew about `Date.prototype.toLocaleString()`, but didn't know it worked for dollar amounts, too!